cmd/upgrade/roles/cmd.go

```diff
@@ -338,18 +338,19 @@ func run(cmd *cobra.Command, argv []string) error {
 		return err
 	}
 
+	if spin != nil {
+		spin.Stop()
+	}
+
 	if len(missingRolesInCS) <= 0 && !isOperatorPolicyUpgradeNeeded {
+		r.Reporter.Infof(
+			"Operator roles/policies associated with the cluster '%s' are already up-to-date.",
+			cluster.ID(),
+		)
 		if !args.isInvokedFromClusterUpgrade {
```

Comment:
NP 
Not directly related to this MR, but can simplify:
```suggestion
		if args.isInvokedFromClusterUpgrade {
		    return nil
		}
		os.Exit(0)
```
WDYT?